node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java

```diff
@@ -761,6 +761,48 @@ public void clearProcessParameter(
     }
   }
 
+  @CommandMethod("tasks task <name> unset javaCommand")
+  public void unsetJavaCommand(
+    @NonNull CommandSource source,
+    @NonNull @Argument("name") Collection<ServiceTask> serviceTasks
+  ) {
+    for (var serviceTask : serviceTasks) {
+      this.updateTask(serviceTask, builder -> builder.javaCommand(null));
+      source.sendMessage(I18n.trans("command-tasks-set-property-success",
```

Comment:
I dont like this playo. This looks like a lot of duplicate code :( but if klaro likes it then it's okay.

Reply:
How would you like to do it?

Reply:
I think it's about moving the for loop into a seperate method, and I can see the issue. Probably best to move into another pull request tho? @robertschuck